addons/mail/models/mail_thread.py

```diff
@@ -1623,11 +1623,13 @@ def _message_add_suggested_recipient(self, result, partner=None, email=None, lan
         if partner and partner.id in [val[0] for val in result[self.ids[0]]]:  # already existing partner ID -> skip
             return result
         if partner and partner.email:  # complete profile: id, name <email>
-            result[self.ids[0]].append((partner.id, partner.email_formatted, lang, reason))
+            result[self.ids[0]].append((partner.id, partner.email_formatted, lang, reason, {}))
```

Comment:
It seems there is a mismatch between commits: default_create_value is added in docstring in first commit (allows model to provide partner default values) but actually implemented in the third ( prefill partner creation form with current record data) ^^

Reply:
Actually, I put that part in that commit because it was directly related to the javascript modification (data transmitted by _get_mail_thread_data).

Reply:
Ah ok :)